CHANGES.rst

```diff
@@ -3,7 +3,86 @@
 0.12 (unreleased)
 -----------------
 
-- Planned for May 15, 2019
+Summary
++++++++
+
+- Released on May 30, 2019
+- 9 contributors
+- 2 months of work
+- 63 pull requests (not all listed below)
+
+**What's new**
+
+For Gammapy v0.12 we did our homework, cleaned-up the basement and emptied the
+trash bin. It is a smaller release that does not introduce many new features,
+but where we have put a lot of effort into integrating the ``gammapy.spectrum``
+submodule into the datasets framework we introduced in the previous Gammapy version.
+For this we replaced the former ``SpectrumObservation`` class by a new ``SpectrumDatasetOnOff``
+class, which now works with the general ``Fit`` and ``Datasets`` objects in
+``gammapy.utils.fitting``. This also enabled us to remove the ``SpectrumObservationList``
+and ``SpectrumFit`` abstractions. We adapted the ``SpectrumExtraction`` class accordingly.
+If you need help with transitioning your old spectral analysis scripts, please let us know.
+We also refactored the ``NDData`` class to use ``MapAxis`` to handle the data axes. This
+affects the ``CountsSpectrum`` and IRF classes in ``gammapy.irf``.
+
+In addition we changed the ``FluxPointEstimator`` to work with the new ``SpectrumDatasetOnOff``
+as well as the ``MapDataset``. Now it is possible to compute flux points for 1D
+as well 3D data with a uniform API. We added a new ``NaimaModel`` wrapper class, which allows
+to fit true physical, spectral models directly to counts based gamma-ray data. To improve
+the fit convergence of the ``SkyDisk`` and ``SkyEllipse`` models we introduced a new
+parameter defining the slope of the edge of these models.
+
+**Contributors:**
+
+In alphabetical order by first name:
+
+- Atreyee Sinha
+- Axel Donath
+- Christoph Deil
+- Dirk Lennarz
+- Debanjan Bose (new)
+- José Enrique Ruiz
+- Lars Mohrmann
+- Luca Giunti
+- Régis Terrier
+
+Pull Requests
++++++++++++++
+-
+- [#2171] Remove Poisson chi2 approximations (Christoph Deil)
+- [#2169] Remove warning astropy_helpers.sphinx.conf is deprecated (José Enrique Ruiz)
+- [#2166] Remove PHACountsSpectrumList class (Régis Terrier)
+- [#2163] Fix `integrate_spectrum` for small integration ranges (Axel Donath)
```

Comment:
I think you need to either remove the backticks, or put double-backticks, to avoid a Sphinx warning (at least in nitpicky mode)?

Reply:
Done